CHANGELOG.rst

```diff
@@ -17,7 +17,7 @@ x.x.x
 * Add support for ap-east-1 region (Hong Kong)
 * Add possibility to specify instance type to use when building custom AMIs with ``pcluster createami``
 * Speed up cluster creation by having compute nodes starting together with master node
-* Enable ASG CloudWatch metrics for the ASG managing compute nodes
```

Comment:
Minor: It seems we are avoiding the final "dot" in the changelog rows

Reply:
removed!